magento/pwa-studio
# [feat] Always scroll to top on router change

## Description

On router changes, the page should always scroll to top.

This also removes prior attempts to manage scrolling-to-top per route. This should be a universal behavior.

## Related Issue
Closes #2289.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go to any page.
2. Scroll down.
3. Click a link.
4. The page should scroll up to top.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
